static/app/views/settings/project/server-side-sampling/modals/recommendedStepsModal.tsx

```diff
@@ -130,22 +129,16 @@ export function RecommendedStepsModal({
   return (
     <Fragment>
       <Header closeButton>
-        <h4>{t('Recommended next steps\u2026')}</h4>
+        <h4>{t('Next steps')}</h4>
       </Header>
       <Body>
         <List symbol="colored-numeric">
           {!!recommendedSdkUpgrades.length && (
             <ListItem>
-              <h5>{t('Update the following SDK versions')}</h5>
+              <h5>{t('Update your SDK(s) before defining sample rules')}</h5>
```

Comment:
The copy they suggested from a UX perspective is wrong, so lets back the original copy `Update the following SDK versions`